src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/get_interior_vehicle_data_request.cc

```diff
@@ -108,7 +108,7 @@ void GetInteriorVehicleDataRequest::FilterDisabledModuleData(
   if (module_data.keyExists(message_params::kHdRadioEnable) &&
       module_data[message_params::kHdRadioEnable].asBool() == false) {
     module_data.erase(message_params::kHdChannel);
-    module_data.erase(message_params::kAvailableHDs);
```

Comment:
This line should be kept in addition with the use of the new parameter. The availableHds functionality should be supported until it is removed from the API.

Reply:
@JackLivio Thanks. Fixed in be2dd5badfc9fef8734b56fce1a1827f8ac1aee0